### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by menshikh-iv**

### CODE REVIEW

**Comment by souravsingh** [comment]
@menshikh-iv I believe that the tests do not use ``nose`` to run. It requires ``pytest`` to execute.

Also, the errors are attributed to the ``dataset`` module.

**Comment by menshikh-iv** [comment]
@souravsingh  about failed tests, I see why this broken, but I didn't understand how to fix it (I disable SSL check with env var, but this doesn't work)
About `pytest` - not critical here (problem with failed tests right now).

Can you look into this problem `urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)>`, how I can reproduce it on laptop? Why `PYTHONHTTPSVERIFY` doesn't work?

**Comment by souravsingh** [comment]
The `PYTHONHTTPSVERIFY` should work in this scenario. I will try to reproduce the error in Linux system, but I think it could a spontaneous failure.

**Comment by souravsingh** [comment]
I am not seeing the error while building. @jakirkham Can you help with the patch?

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ jobs:
     machine: true
     environment:
       - CONDA_PY: "27"
+      - PYTHONHTTPSVERIFY: 0
```
Comment (jakirkham) [inline .circleci/config.yml]: Editing CI files directly is strongly discouraged. Please use `conda-forge.yml`'s `matrix` feature if you want to do this and re-render. Here's an example ( https://github.com/conda-forge/tk-feedstock/pull/20 ). Though honestly I don't understand why this is needed. 😕

**Comment by jakirkham** [comment]
Ah this might be related to a `conda-build` bug. Would try the [suggestion in this comment]( https://github.com/conda-forge/mosfit-feedstock/issues/23#issuecomment-349016207 ).

**Comment by menshikh-iv** [comment]
replaced by #9

**Pull request closed without merging**